<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>userman_work</class>
 <widget class="QMainWindow" name="userman_work">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>682</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Êù®ÊñáË¢ú‰∏öüôåÁÆ°ÁêÜÁ≥ªÁªü V3.0</string>
  </property>
  <property name="windowIcon">
   <iconset resource="src.qrc">
    <normaloff>:/socks_icon_1.png</normaloff>:/socks_icon_1.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btn_excel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>340</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>ÂØºÂá∫</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_plus">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>160</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Â¢ûÂä†</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_search">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>20</y>
      <width>311</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTableWidget" name="tab_sql">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>70</y>
      <width>521</width>
      <height>371</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>760</width>
      <height>461</height>
     </size>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="showGrid">
     <bool>true</bool>
    </property>
    <column>
     <property name="text">
      <string>ÂßìÂêç</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>ÁîµËØù</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>ÂØÜÁ†Å</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>ÊùÉÈôê</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="btn_exit">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>400</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>ÈÄÄÂá∫</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_update">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>280</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Âà∑Êñ∞</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>111</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="src.qrc">:/images.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_select">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>20</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Á≠õÈÄâ</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_clear">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>20</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Ê∏ÖÁ©∫</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_mod">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>220</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>‰øÆÊîπ</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>682</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="src.qrc"/>
 </resources>
 <connections/>
</ui>
